paddle/fluid/pir/dialect/operator/interface/infer_symbolic_shape/unary_infer_sym.cc

```diff
@@ -246,6 +246,81 @@ bool DiagonalOpInferSymbolicShape(
   return true;
 }
 
+bool DistributeFpnProposalsOpInferSymbolicShape(
+    pir::Operation *op, pir::InferSymbolicShapeContext *infer_context) {
+  const auto &attributes = op->attributes();
+  int32_t min_level =
+      attributes.at("min_level").dyn_cast<pir::Int32Attribute>().data();
+  int32_t max_level =
+      attributes.at("max_level").dyn_cast<pir::Int32Attribute>().data();
+  int32_t num_levels = max_level - min_level + 1;
+
+  symbol::DimExpr num_rois = [&]() {
+    pir::Value rois_num = op->operand_source(1);
+    const auto &rois_num_shape_or_data =
+        infer_context->GetShapeOrDataForValue(rois_num);
+
+    PADDLE_ENFORCE_EQ(
+        rois_num_shape_or_data.shape()[0],
+        1,
+        phi::errors::InvalidArgument("DistributeFpnProposalsOp in pir model "
+                                     "only support batch_size=1 now."));
+
+    PADDLE_ENFORCE_EQ(rois_num_shape_or_data.data().has_value(),
+                      true,
+                      phi::errors::InvalidArgument(
```

Comment:
```suggestion
                      ::common::errors::InvalidArgument(
```
后续也可以注意下，更推荐使用common